libs/shared/utils/src/lib/createXRoadAPIPath.ts

```diff
@@ -1,5 +1,3 @@
-import { logger } from '@island.is/logging'
-
```

Comment:
Could the logging module be monkey-patched out instead? We will likely run into more logger issues in the future, similar to this.

Reply:
I don't know about that. I kind of avoid monkey-patching ;)

Reply:
I want to get rid of this function, but in another branch we changed it to use console.error instead to get rid of this dependency.